bundles/ui/org.openhab.ui.webapp/web/WebApp/Action/Logic.js

```diff
@@ -1713,7 +1731,151 @@ var WebApp = (function() {
 			}
 		}
 	}
-
+	
+	function Slider(item) {
+		var _t = this;
```

Comment:
Style: Well, yes, this scripts var names look like so, but wouldn't it be nice to start new functions with more expressive var names? Especially when it's getting minified, it wouldn't make a difference.